tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/BulkheadRapidRetry55MethodSynchBean.java

```diff
@@ -25,13 +25,12 @@
 import javax.enterprise.context.ApplicationScoped;
 
 import org.eclipse.microprofile.fault.tolerance.tck.bulkhead.Utils;
-import org.eclipse.microprofile.faulttolerance.Asynchronous;
 import org.eclipse.microprofile.faulttolerance.Bulkhead;
 import org.eclipse.microprofile.faulttolerance.Retry;
 import org.eclipse.microprofile.faulttolerance.exceptions.BulkheadException;
 
 /**
- * A simple method level Asynchronous @Bulkhead bean that has a retry option.
```

Comment:
Why did you change the purpose of this test?

Reply:
None of the tests I've edited are used by any asynchronous test - they're used exclusively by _BulkheadSynchRetryTest_. I changed the class descriptions because they're wrong - they've presumably been copied over from other Asynchronous tests without adjusting, or were refactored incorrectly.

Reply:
It's even in the class name :)

Reply:
The class name was wrong! The test itself is to tests async aspect. The correct fix is to rename the class rather than repurpose the test. Otherwise, there is no async test left.

Reply:
? Yes there is: https://github.com/eclipse/microprofile-fault-tolerance/blob/master/tck/src/main/java/org/eclipse/microprofile/fault/tolerance/tck/bulkhead/clientserver/BulkheadRapidRetry55MethodAsynchBean.java

It's almost identical except the linked async bean isn't application scoped

Reply:
thanks @Pandrex247 !